processing/src/main/java/io/druid/segment/incremental/IncrementalIndexStorageAdapter.java

```diff
@@ -532,14 +530,13 @@ public Long get()
                 final Integer metricIndexInt = index.getMetricIndex(column);
                 if (metricIndexInt != null) {
                   final int metricIndex = metricIndexInt;
-
-                  final ComplexMetricSerde serde = ComplexMetrics.getSerdeForType(index.getMetricType(column));
+                  final Class classOfObject = index.getMetricClass(column);
```

Comment:
Isn't it better to return `Float.class` rather than throwing NPE for `classOfObject()`?